src/main/scala/edu/berkeley/ce/rockslicing/CommandReader.scala

```diff
@@ -0,0 +1,13 @@
+package edu.berkeley.ce.rockslicing
+
+/** A simple tool for parsing command line inputs
+  * @constructor Create a new command reader
+  */
+case class CommandReader(inputs: Array[String]) {
```

Comment:
Here's a Stack Overflow post you might want to check out: http://stackoverflow.com/questions/2315912/scala-best-way-to-parse-command-line-parameters-cli. I especially like the second answer.